python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py

```diff
@@ -751,7 +763,35 @@ def _degree(self, vertex_subset, direction=Direction.ALL):
 
         return df
 
-    def to_directed(self, DiG):
+    def make_plc_graph(self, value_col=None, store_transposed=False):
+        if value_col is None:
+            value_col = cudf.Series(
+                cupy.ones(len(self.edgelist.edgelist_df), dtype='float32')
```

Comment:
I like that you added the edge values column for edge lists that don't

Reply:
well, I believe the plc graph creation fails otherwise...

Reply:
Right. So far we have been luck to process datasets that have edge values